compiler/testData/codegen/box/annotations/annotationsOnLateinitAccessors.kt

```diff
@@ -0,0 +1,33 @@
+// WITH_REFLECT
+// IGNORE_BACKEND: JVM_IR
+// TARGET_BACKEND: JVM
+
+// Please make sure that this test is consistent with the diagnostic test "annotationsTargetingLateinitAccessor.kt"
+
+import kotlin.reflect.KAnnotatedElement
+import kotlin.reflect.KProperty
+
+annotation class Ann
+annotation class AnnRepeat
```

Comment:
Looks like this annotation is not used here and in the corresponding diagnostic test, let's remove it

Reply:
Good catch, done.